lua/lvim/core/luasnip/init.lua

```diff
@@ -0,0 +1,59 @@
+local M = {}
+function M.config()
+  lvim.builtin.luasnip = {
+    sources = {
+      friendly_snippets = true,
```

Comment:
```suggestion
      friendly_snippets = {
      	paths = {
      		utils.join_paths(get_runtime_dir(), "site", "pack", "packer", "start", "friendly-snippets")
      	}
      },
```

Reply:
This will not work. friendly_snippets is treated as a boolean in so many places that doing this can be considered a breaking change.
Also, I think it will confuse users because, if they make use of the LSP to explore the config options and see this is a table, it will not be that obvious how to disable it.
I think it is safer to keep it like it is right now and seek for improvements in future refactors